simulation/traffic_simulator/include/traffic_simulator/hdmap_utils/hdmap_utils.hpp

```diff
@@ -227,8 +227,10 @@ class HdMapUtils
   using LaneletId = std::int64_t;
 
   auto isTrafficLight(const LaneletId) const -> bool;
-  auto isTrafficRelation(const LaneletId) const -> bool;
-  auto getTrafficRelation(const LaneletId) const -> lanelet::TrafficLight::Ptr;
+  auto isTrafficLightRelation(const LaneletId) const -> bool;
+  auto getTrafficLightRelation(const LaneletId) const -> lanelet::TrafficLight::Ptr;
+
+  auto getTrafficLightRelationIDsFromWayID(const LaneletId) const -> std::vector<LaneletId>;
```

Comment:
The term `traffic light relation` probably does not exist in lanelets. Also, this function handles "relation representing traffic light", not "relation of traffic light". Change the function name (to a name that is neither the current name nor the original name).

Reply:
I decided to use `traffic light regulatory element` instead of `traffic light relation`.
The word "traffic light" does not exist in the concept of relation in OpenStreetMap, and seems to have been introduced in the "regulatory element" of lanelet2.
That's why I decided to stop referring to "relation" and using "regulatory element" in the naming.

Reply:
the re-naming was done in https://github.com/tier4/scenario_simulator_v2/pull/1027/commits/f4a0c2adcc94185dd43839af3a81d25dba0783c7